python/lsst/ts/logging_and_reporting/web_app/services/scheduler_service.py

```diff
@@ -0,0 +1,768 @@
+                    data['fill_alpha'][k] = 0.5;
+                    data['line_alpha'][k] = Math.max(
+                        0,
+                        1 - (mjd_val - data['mjd'][k]) / scale
+                    );
+                } else {
+                    data['fill_alpha'][k] = 0.0;
+                    data['line_alpha'][k] = 0.0;
+                }
+            }
+            cds.change.emit();
+        }
+    }
+
+    // Update sun and moon visibility
+    if (current_day === null) {
+        day_label.text = "";
+        _hide_all_celestial_markers(all_sun_markers, all_moon_markers);
+    } else {
+        const idx = day_obs_list.indexOf(current_day);
+        _update_celestial_markers(all_sun_markers, all_moon_markers, idx);
+        day_label.text = "Night: " + current_day;
+    }
+
+    function _hide_all_celestial_markers(sun_markers, moon_markers) {
+        for (let s = 0; s < sun_markers.length; s++) {
+            for (let m = 0; m < sun_markers[s].length; m++) {
+                sun_markers[s][m].glyph.fill_alpha = 0.0;
+                moon_markers[s][m].glyph.fill_alpha = 0.0;
+                sun_markers[s][m].data_source.change.emit();
+                moon_markers[s][m].data_source.change.emit();
+            }
+        }
+    }
+
+    function _update_celestial_markers(sun_markers, moon_markers, night_idx) {
+        for (let s = 0; s < sun_markers.length; s++) {
+            for (let m = 0; m < sun_markers[s].length; m++) {
+                if (m === night_idx) {
+                    sun_markers[s][m].glyph.fill_alpha = 1.0;
+                    moon_markers[s][m].glyph.fill_alpha = 0.8;
+                } else {
+                    sun_markers[s][m].glyph.fill_alpha = 0.0;
+                    moon_markers[s][m].glyph.fill_alpha = 0.0;
+                }
+                sun_markers[s][m].data_source.change.emit();
+                moon_markers[s][m].data_source.change.emit();
+            }
+        }
+    }
+    """
+
+    mjd_slider.js_on_change(
+        "value",
+        bokeh.models.CustomJS(
+            args=dict(
+                mjd_slider=mjd_slider,
+                sources=night_renderers,
+                day_label=dayobs_label,
+                day_obs_list=unique_nights,
+                mjd_starts=[cond.sun_n12_setting for cond in conditions_list],
+                mjd_ends=[cond.sun_n12_rising for cond in conditions_list],
+                scale=fade_scale,
+                all_sun_markers=all_sun_markers,
+                all_moon_markers=all_moon_markers
+            ),
+            code=callback_code
+        )
+    )
+
+def create_visit_skymaps(
+    visits,
+    nside=32,
+    observatory=None,
+    timezone="Chile/Continental",
+    planisphere_only=False,
+    applet_mode=False,
+    theme="LIGHT",
+):
+    """
+    Prepare data for multi-night SphereMap plotting.
+    Returns figure and data dict.
+    This is a modified version of schedview.plot.visitmap.create_visit_skymaps
+    """
```

Comment:
could you complete this docstring with the Parameters and Return section 🙏 ? I see you didn't do that for private methods, but you did for the `plot_visit_skymaps` one.